script.js

```diff
@@ -1,4 +1,175 @@
-  return myOutputValue;
+  //prompt player 1 to draw a card
+  if (gamePhase == gamePhase_starting_game) {
+    gamePhase = gamePhase_draw_first_card;
+    return ` player, please draw a card by clicking "submit"`;
+    //player 1 has drawn 1st card
+  } else if (gamePhase == gamePhase_draw_first_card) {
+    gamePhase = gamePhase_draw_second_card;
+    var playerCard1 = drawCard();
+    playerCards.push(playerCard1);
+    //computer automatically draws 1st card
+    var computerCard1 = drawCard();
+    computerCards.push(computerCard1);
+    return `Player you drew ${JSON.stringify(
+      playerCards
+    )}, please draw another card by clicking "submit"`;
+    //player 1 to draw 2nd card
+  } else if (gamePhase == gamePhase_draw_second_card) {
+    gamePhase = gamePhase_hit_or_stand;
+    var playerCard2 = drawCard();
+    playerCards.push(playerCard2);
+    //computer to automatically draw 2nd card
+    var computerCard2 = drawCard();
+    computerCards.push(computerCard2);
+    return `Player you now have ${JSON.stringify(
+      playerCards
+    )}, with a total sum of ${calculateSum(
+      playerCards
+    )}, would you like to hit or stand?`;
+    //outcome when player decides to hit or stand
+  } else if (gamePhase == gamePhase_hit_or_stand) {
+    //if player wants to hit, draw 1 additional card
+    if (input == "hit") {
+      gamePhase = gamePhase_draw_card;
+      return `Please click to draw a card`;
+      // if player wants to stand, stop drawing
+    } else if (input == "stand") {
+      gamePhase = gamePhase_stop_drawing;
+      return `You stand, please click to continue`;
+    }
+    //If player keeps inputting "hit", keep drawing card and adding the total sum
+  } else if (gamePhase == gamePhase_draw_card) {
+    gamePhase = gamePhase_hit_or_stand;
+    var playerNewCard = drawCard();
+    playerCards.push(playerNewCard);
+    console.log(gamePhase, "gamephase");
+    return `Player you now have ${JSON.stringify(
+      playerCards
+    )}, with a total sum of ${calculateSum(
+      playerCards
+    )}, would you like to hit or stand?`;
+  } else if (gamePhase == gamePhase_stop_drawing) {
+    //If sum of computer hands<17, keep drawing a card and add to card array
+    while (calculateSum(computerCards) < 17) {
+      computerCards.push(drawCard());
+    }
+    //compute player hand
+    var playerSum = calculateSum(playerCards);
+    //computer computer hands
+    var computerSum = calculateSum(computerCards);
+    //compare the winning outcome
+    var gameOutcome = "";
+    if (playerSum > 21 && computerSum > 21) {
+      gameOutcome = "draw";
+    } else if (playerSum <= 21 && computerSum > 21) {
+      gameOutcome = "win";
+    } else if (playerSum > 21 && computerSum <= 21) {
+      gameOutcome = "lose";
+    } else if (playerSum < computerSum) {
+      gameOutcome = "lose";
+    } else if (computerSum < playerSum) {
+      gameOutcome = "win";
+    } else {
+      gameOutcome = "draw";
```

Comment:
good job being able to implement the logic for the game. However, if we look at this section of code, the if statement that you are implementing stretches for nearly 70 lines, which is a bit too long for an if statement and a function. In this scenario, it would be preferable if you used a switch statement instead and place the logic within your larger else ifs into smaller helper functions. 

So what I mean by this is that we can use a switch statement for the different values of your gamePhase variables and for the code logic within it, you can separate them out into a helper function if the logic is more complex.